lib_xua/src/core/audiohub/xua_audiohub.xc

```diff
@@ -660,296 +737,324 @@ void XUA_AudioHub(chanend ?c_aud, clock ?clk_audio_mclk, clock ?clk_audio_bclk,
     unsigned command;
     unsigned mClk;
     unsigned divide;
+    unsigned audioActive = 1; /* Flag used to indicate whether both interfaces are set to Alt 0 or not for power saving option
```

Comment:
As discussed, initialising it to 0 would handle going from enumerated -> idle instead of enumerated -> streaming

Reply:
Done - turned out to be a very fiddly change but now implemented and passing tests